OpenSim/Common/ComponentConnector.h

```diff
@@ -446,17 +491,9 @@ class OSIMCOMMON_API AbstractInput : public AbstractConnector {
             channelName = "";
         }
         
-        // Annotation.
-        if (leftParen != std::string::npos && rightParen != std::string::npos) {
-            annotation = connecteeName.substr(leftParen + 1,
-                                              rightParen - (leftParen + 1));
-        } else {
-            if (!channelName.empty()) {
-                annotation = channelName;
-            } else {
-                annotation = outputName;
-            }
-        }
+        // Alias.
+        alias = connecteeName.substr(leftParen + 1, rightParen - (leftParen + 1));
```

Comment:
The checks for `npos` have not been removed. I have attached a screencap of the diff I see when I click "Files changed" at the top of the PR. The checks are also present when I pull the branch.

![diff](https://cloud.githubusercontent.com/assets/4203505/18851266/fe615bac-83ef-11e6-92ee-475bd42f2486.png)